index.html

```diff
@@ -8,10 +8,26 @@
     <title>Awesome Books</title>
   </head>
   <body>
-    <div class="center">
-      <h1>All awesome books</h1>
-      <div class="container"></div>
-      <h2>Add new books</h2>
+    <header>
+      <nav>
+          <div class="container-nav-menu">
+              <div class="logo">Awesome Books</div>  
+              <ul class="menu__items">
+                  <li class="menu__items-item">List</li>
+                  <li class="menu__items-item">Add New</li>
+                  <li class="menu__items-item">Contact</li>
+              </ul>
+          </div>
+      </nav>
+  </header>
+  <div class="list-list">
+  <h1 class="head">All awesome books</h1>
+  <div class="container">
+  </div>
```

Comment:
- [ OPTIONAL ] I noticed a minor bug in your project. I have attached a video reference below. I noticed that whenever a new book was added and I switch to the list of books page, It showed all right but then when the page is refreshed nothing shows on the page. However, when I switch from that page to either the contacts page or the Add book page and switch back, the book shows. So kindly fix this so users don't get confused when using your app :+1: 

[Screencast from 04-26-2023 01:17:11 PM.webm](https://user-images.githubusercontent.com/102699958/234587512-8cad947c-86b6-40f4-b633-a4cb9a393946.webm)

Reply:
well resolved! ✔️